tests/resources/conftest.py

```diff
@@ -33,11 +33,11 @@ def record_resource(service):
     return RecordResource(CustomRecordResourceConfig, service)
 
 
-@pytest.fixture(scope="module")
-def base_app(base_app, record_resource):
-    """Application factory fixture."""
-    base_app.register_blueprint(record_resource.as_blueprint())
-    yield base_app
```

Comment:
By relying on entry points to load blueprints, this kind of thing is not needed anymore.